### DISCUSSION TITLE
Better data than SRTM?

### DISCUSSION THREAD

**Discussion by stevempotter** [domlysz/BlenderGIS#532] [Q&A]
Will you be adding data from other sources that is newer (SRTM30 is from 2000) or higher resolution?

For working with maps that are only a few tens of km across, 30m (1 arc-second) is not very good.

**Comment by domlysz** [comment]
SRTM is a free and worldwide dataset. Higher resolution datasource avaibility depends of your area of interest and can be expensive. But if you have access to this kind of data in geotiff format, you can import it in Blender with the addon.
